resources/rpm/project.spec.erb

```diff
@@ -1,4 +1,4 @@
-%global debug_package %{nil}
+%define debug_package %{nil}
```

Comment:
we should make sure this doesn't break things for other platforms, I feel like this was super finicky in the past :\

specifically, the build may work but we should check and make sure the package sizes haven't changed dramatically